Vitals/VitalValueTester.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Diagnostics;
+using static Vitals.VitalStatusLogger;
+namespace Vitals
+{
+    /// <summary>
+    /// This class checks every vital level
+    /// and triggers warning in case of any breach
+    /// </summary>
+    class VitalValueTester
+    {
+        public static bool CheckValueAgainstRange(string vital, float value, int least, int? highest)
+        {
+            if (value < least)
+            {
+                LogBreach(vital, "low", value);
+                return false;
+            }
+            else if (upperLimitTrigger(value, highest))
+            {
+                LogBreach(vital, "high", value);
+                return false;
+            }
+            return true;
```

Comment:
Any reason not to call LogVitalsOk here? Currently the responsibility to trigger the log is spread across two classes. This kind of distribution tends towards a 'non-cohesive' structure.

Reply:
If I call LogVitalsOk there, then it logs 3 times( once for each vital). I thought it makes more sense to log just once.  Should I try to find a way around it?

Reply:
Good point. I didn't reason it out, but now I understand.